### PR TITLE
Check the study UID in Orthanc Raw is the same as in VNA / PACS [force-system-test]

### LINKED ISSUE #496: Orthanc Raw modifies `StudyInstanceUID`, `SeriesInstanceUID`, and `SOPInstanceUID` when modifying the study project name tag`
I might be doing something wrong but I think the `StudyInstanceUID`, and `SeriesInstanceUID`, and `SOPInstanceUID` values are modified by Orthanc Raw when retrieving data from the archives, which means we can't compare `SOPInstanceUID`s to check for existing instances. I'll open a PR with a test to check the `StudyInstanceUID` in Orthanc Raw is the same as in the VNA (assuming this is the expected behaviour?)

_Originally posted by @p-j-smith in https://github.com/SAFEHR-data/PIXL/issues/486#issuecomment-2326050577_

### PR DESCRIPTION
**Pull request by p-j-smith**
## Description
Related to #486.
- For #486, we need to compare the `SOPInstanceUID`s in Orthanc Raw against those in the VNA / PACS. However, when retrieving data, the `StudyInstanceUID`, and `SeriesInstanceUID`, and `SOPInstanceUID` values are modified by Orthanc Raw.

Fixes #496 
- keep `StudyInstanceUID`, `SeriesInstanceUID`, and `SOPInstanceUID` when setting the project name private tag in Orthanc Raw. Also set `Force=True` to allow us to keep these UIDs
- set `OverwriteInstance: True` in the PIXL Raw config so that if an instance with an existing SOPInstanceUID is uploaded, the previous instance is overwritten (otherwise we'd need to manually delete the old instance after setting the project name tag)
- Add test to check the `StudyInstanceUID`, `SeriesInstanceUID`, and `SOPInstanceUID` are correct in Orthanc raw after retrieving a study and setting the project name tag
- update the config for Orthanc Raw in the imaging API tests to be the same as the production config

## Type of change
Please delete options accordingly to the description.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Suggested Checklist

- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have passed on my local host device. (see further details at the [CONTRIBUTING](https://github.com/SAFEHR-data/PIXL/blob/main/CONTRIBUTING.md#local-testing) document)
- [ ] Make sure your branch is up-to-date with main branch. See [CONTRIBUTING](https://github.com/SAFEHR-data/PIXL/blob/main/CONTRIBUTING.md) for a general example to syncronise your branch with the `main` branch.
- [ ] I have requested PR review to `UCLH-Foundtry/arc-dev`
- [ ] I have adddressed and marked as resolved all the review comments in my PR.
- [ ] Finally, I have selected `squash and merge`

### CODE REVIEW

File: pixl_imaging/tests/docker-compose.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ services:
             context: ../../
             dockerfile: ./docker/orthanc-raw/Dockerfile
             args:
-                PIXL_DICOM_TRANSFER_TIMEOUT: 20
+                PIXL_DICOM_TRANSFER_TIMEOUT: 30
```
Comment (p-j-smith) [inline pixl_imaging/tests/docker-compose.yml:73]: just to be consistent with the system tests, happy to change it back

File: pixl_imaging/src/pixl_imaging/_orthanc.py [hunk 2]
```diff
@@ -93,10 +93,11 @@ async def modify_private_tags_by_study(
             f"/studies/{study_id}/modify",
             {
                 "PrivateCreator": private_creator,
-                "Permissive": False,
-                "KeepSource": False,
+                "Permissive": True,
```
Comment (p-j-smith) [inline pixl_imaging/src/pixl_imaging/_orthanc.py:96]: A 400 error is raised if `Permissive: False` - not sure if this is an issue?

**Comment by p-j-smith** [comment]
I'm not sure why the system tests are being [skipped](https://github.com/SAFEHR-data/PIXL/actions/runs/10684042479/job/29614139672?pr=495)? I tried rerunning them but they were skipped again

**Comment by mxochicale** [comment]
> I'm not sure why the system tests are being [skipped](https://github.com/SAFEHR-data/PIXL/actions/runs/10684042479/job/29614139672?pr=495)? I tried rerunning them but they were skipped again

Does this tests need to be passing locally by going to (tests and then `./run-system-test.sh`)? As I tried and I have this log:
```
2024-09-04 16:07:54.866 | ERROR    | system_test:_check_dcm_tags_from_zip:179 - TEMPORARILY IGNORE: tag value b'test-extract-uclh-omop-cdm' should be of type str, but is of type bytes
============================================================================================= short test summary info =============================================================================================
FAILED system_test.py::TestFtpsUpload::test_ftps_dicom_upload - assert {('ANONYMIZED', 'AP')} == {('ANONYMIZED...'include123')}
===================================================================================== 1 failed, 4 passed in 374.44s (0:06:14) =====================================================================================
(pixlVE) mxochicale@precision-5570:~/Desktop/saferHR/repositories/PIXL/test$ ./run-system-test.sh stop

```

**Comment by p-j-smith** [comment]
> Does this tests need to be passing locally by going to (tests and then ./run-system-test.sh)? As I tried and I have this log:

> 2024-09-04 16:07:54.866 | ERROR    | system_test:_check_dcm_tags_from_zip:179 - TEMPORARILY IGNORE: tag value b'test-extract-uclh-omop-cdm' should be of type str, but is of type bytes
============================================================================================= short test summary info =============================================================================================
FAILED system_test.py::TestFtpsUpload::test_ftps_dicom_upload - assert {('ANONYMIZED', 'AP')} == {('ANONYMIZED...'include123')}
===================================================================================== 1 failed, 4 passed in 374.44s (0:06:14) =====================================================================================
(pixlVE) mxochicale@precision-5570:~/Desktop/saferHR/repositories/PIXL/test$ ./run-system-test.sh stop

Oh that's [this](https://github.com/SAFEHR-data/PIXL/issues/4) issue, which I thought would be fixed as part of https://github.com/SAFEHR-data/PIXL/pull/489 😢 Perhaps we should further increase `PIXL_DICOM_TRANSFER_TIMEOUT` and `ORTHANC_RAW_EXTRA_STABLE_SECONDS`?

**Comment by mxochicale** [comment]
> Oh that's [this](https://github.com/SAFEHR-data/PIXL/issues/4) issue, which I thought would be fixed as part of #489 😢 Perhaps we should further increase `PIXL_DICOM_TRANSFER_TIMEOUT` and `ORTHANC_RAW_EXTRA_STABLE_SECONDS`?

Thanks, just wanted to do a quick test and tried `PIXL_DICOM_TRANSFER_TIMEOUT=400` and `ORTHANC_RAW_EXTRA_STABLE_SECONDS=400` but seems to produce some errors. I leave logs in case are useful. 

```E       AssertionError: Condition at_least_n_studies_exported was not achieved even after 211 seconds

../pytest-pixl/src/pytest_pixl/helpers.py:111: AssertionError
============================================================================================= short test summary info =============================================================================================
ERROR system_test.py::TestFtpsUpload::test_ftps_parquet_upload - AssertionError: Condition at_least_n_studies_exported was not achieved even after 211 seconds
ERROR system_test.py::TestFtpsUpload::test_ftps_radiology_linker_upload - AssertionError: Condition at_least_n_studies_exported was not achieved even after 211 seconds
ERROR system_test.py::TestFtpsUpload::test_ftps_dicom_upload - AssertionError: Condition at_least_n_studies_exported was not achieved even after 211 seconds
ERROR test_parquet_exports.py::test_public_parquet - AssertionError: Condition at_least_n_studies_exported was not achieved even after 211 seconds
===================================================================================== 1 passed, 4 errors in 357.37s (0:05:57)
```

**Review by stefpiatek — APPROVED** [review]
Ah so nice to have this solved, thanks Paul

**Comment by stefpiatek** [comment]
> Thanks, just wanted to do a quick test and tried `PIXL_DICOM_TRANSFER_TIMEOUT=400` and `ORTHANC_RAW_EXTRA_STABLE_SECONDS=400` but seems to produce some errors. I leave logs in case are useful.

Ah this would be because the test waits 120 seconds for a study to be expected, but it will take 800 seconds (`PIXL_DICOM_TRANSFER_TIMEOUT` + `ORTHANC_RAW_EXTRA_STABLE_SECONDS`) before it will move from orthanc-raw to orthanc-anon

**Pull request merged**